android/app/src/main/java/org/fossasia/openevent/fragments/DayScheduleFragment.java

```diff
@@ -83,36 +88,37 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle sa
         Utils.registerIfUrlValid(swipeRefreshLayout, this, this::refresh);
         setUpRecyclerView();
 
-        if (savedInstanceState != null && savedInstanceState.getString(SEARCH) != null) {
-            searchText = savedInstanceState.getString(SEARCH);
-        }
-
-        realmResults = realmRepo.getSessionsByDate(date, SortOrder.sortTypeSchedule());
-        realmResults.addChangeListener((sortedSessions, orderedCollectionChangeSet) -> {
-            sessions.clear();
-            sessions.addAll(sortedSessions);
-
-            filteredSessions.clear();
-            filteredSessions.addAll(sortedSessions);
+        dayScheduleFragmentViewModel = ViewModelProviders.of(this)
+                .get(DayScheduleFragmentViewModel.class);
```

Comment:
No need to add it to a new line. We don't need to strictly follow the 100 character limit, if it is not affecting the readability of the code i.e. user doesn't need to scroll left/right.